### ISSUE TITLE
Compile Markdown includes within kit templates

### ISSUE THREAD

**Issue by jeffwhitfield** [thehive/codekit-kickoff#5]
Hi there!

Downloaded and played around with Codekit Kickoff. Really like the simplicity and ease of use compared to using more complex Grunt-based static site generators. However, I'm having one issue...

How would I go about including Markdown documents that get rendered as HTML upon build? Tried to do just a simple include or import but no luck. Is there a trick to this? Would be really cool to have this since it allows you to quickly stub in content delivered from clients in a Markdown file.  :)

**Comment by thehive** [comment]
Hey Jeff,

Glad you like the Kickoff. I understand your concern around Markdown. Out of the box, my project doesn’t include any pointers to MD -> HTML, but it is possible with CodeKit. Ultimately, if you were to create an MD file in your project, you could just update how you want it to be managed (output, etc.) from the CodeKit Settings panel.

https://incident57.com/codekit/help.html#markdown <https://incident57.com/codekit/help.html#markdown>

Hope this helps!

Mike

Michael Lesher
-------------------------------------
mike.lesher@gmail.com
585-202-3902
iamlesher.com
@iamlesher

**Comment by jeffwhitfield** [comment]
Hmm…so basically, Codekit can’t process .kit files into anything but HTML using the @import or @include methods? Heck, if there’s a way to tell it to process a specific filetype upon import then it would open up a LOT of possibilities.  :)

Jeff

**Comment by thehive** [comment]
Yeah - that’s a good point! I would say to reach out to Bryan at CodeKit. He has been pretty responsive to me in the past.

Mike

Michael Lesher
Owner // Lead Software Architect
——————————————————
mike@thehivemedia.com
thehivemedia.com
@iamlesher

**Comment by jeffwhitfield** [comment]
Late reply, I know. Got busy and let the inbox fill up!  :P

I’ll definitely reach out to Bryan soon. Would be really cool to see what we can do to get CodeKit in a place where it could realistically be used as a simple, flexible static site generator.  :D

Jeff Whitfield
Web Architect
jeff@soulcraftgroup.com
214-233-4329
http://www.soulcraftgroup.com <http://www.soulcraftgroup.com/>
